layouts/partials/main-page.html

```diff
@@ -193,6 +41,33 @@ <h2>Introducing New Projects</h2>
       </p>
     </div>
   </div>
+  <div class="preview-section__feature">
+    <div class="preview-section__feature__description right">
+      <h2>Powerful shell underneath</h2>
+      <p>
+        We re-designed the shell powering Rizin from scratch, leveraging Tree-sitter to define and parse the shell
```

Comment:
Maybe worth adding a link to Tree-Sitter site?

Reply:
Also "and parse the shell grammar" -> "and parse the grammar" to avoid tautology.